### DISCUSSION TITLE
academy/node-js/handle-blocked-requests-puppeteer

### DISCUSSION THREAD

**Discussion by giscus** [apify/apify-docs#1075] [Comments]
# academy/node-js/handle-blocked-requests-puppeteer

Getting around website defense mechanisms when crawling

https://docs.apify.com/academy/node-js/handle-blocked-requests-puppeteer

**Comment by tyerobinson** [comment]
Am I the only person getting error stating Did not expect property 'launchPuppeteerOptions/gotoFunction handlePageFunction' to exist errors?

**Reply by tyerobinson** [reply]
meant to say retireInstanceAfterRequestCount handlePageFunction does exist
